src/libraries/System.Private.Xml/src/System/Xml/Dom/XmlElement.cs

```diff
@@ -37,7 +37,7 @@ internal XmlElement(XmlName name, bool empty, XmlDocument doc) : base(doc)
             }
         }
 
-        protected internal XmlElement(string prefix, string localName, string? namespaceURI, XmlDocument doc)
+        protected internal XmlElement(string? prefix, string localName, string? namespaceURI, XmlDocument doc)
```

Comment:
The ref needs to be updated, too.

Also, I see some other occurrences of `string prefix, string localName` in the refs:
```C#
System.Xml.ReaderWriter\ref\System.Xml.ReaderWriter.cs(2652):public virtual void AppendChildElement(string prefix, string localName, string namespaceURI, string value) { }
System.Xml.ReaderWriter\ref\System.Xml.ReaderWriter.cs(2657):public virtual void CreateAttribute(string prefix, string localName, string namespaceURI, string value) { }
System.Xml.ReaderWriter\ref\System.Xml.ReaderWriter.cs(2677):public virtual void InsertElementAfter(string prefix, string localName, string namespaceURI, string value) { }
System.Xml.ReaderWriter\ref\System.Xml.ReaderWriter.cs(2678):public virtual void InsertElementBefore(string prefix, string localName, string namespaceURI, string value) { }
System.Xml.ReaderWriter\ref\System.Xml.ReaderWriter.cs(2713):public virtual void PrependChildElement(string prefix, string localName, string namespaceURI, string value) { }
```

Do any of those need updating as well?

Reply:
Oh, yes, I forgot about the refs. I'm normally reminded by the compiler, but that didn't happen here. The only place I can see what _looks like_ a ref, is in `libraries\System.Xml.ReaderWriter\ref\System.Xml.ReaderWriter.cs`. But that seems like a strange place to have it. Do you know if this is the correct place?

Reply:
>Do any of those need updating as well?

Well spotted, yes they do. I should've checked similar methods when commiting. I checked the usages of those and they call into `public abstract void WriteStartElement(string? prefix, string localName, string? ns);`